includes/admin/class-wp-job-manager-addons-landing-page.php

```diff
@@ -78,17 +79,32 @@ public function add_menu_items() {
 		 *
 		 * @param bool $show True if the menu should be added.
 		 */
-		if ( apply_filters( 'job_manager_addon_upsell_applications', ! get_option( 'job_manager_addon_upsell_applications' ) ) ) {
+		if ( apply_filters( 'job_manager_addon_upsell_applications', ! $was_addon_installed && ! $was_page_dismissed ) ) {
 
 			add_submenu_page(
-				$parent_page,
+				'edit.php?post_type=job_listing',
 				__( 'WP Job Manager - Applications', 'wp-job-manager' ),
-				sprintf( '%s <span class="awaiting-mod wpjm-addon-upsell__badge">%s</span>', __( 'Applications', 'wp-job-manager' ), $badge_text ),
+				sprintf(
+					'%s <span class="awaiting-mod wpjm-addon-upsell__badge">%s</span>',
+					__( 'Applications', 'wp-job-manager' ),
```

Comment:
I think that we need to escape or sanitize this, as it may be an attack vector (
```suggestion
					esc_html__( 'Applications', 'wp-job-manager' ),
```
Or:
```suggestion
					sanitize_text_field( __( 'Applications', 'wp-job-manager' ) ),
```

Reply:
Hm looks like it's not usually escaped in our plugin (or [WC](https://github.com/search?q=repo%3AAutomattic%2Fwoocommerce%20add_submenu_page&type=code)), but wouldn't hurt. It goes through `wptexturize` but doesn't seem to be escaped anywhere before rendering.

I'll open a dedicated PR to update these in in the whole plugin.